### PR TITLE
fix(svd-parser): parse `xs:noNamespaceSchemaLocation`

### PR DESCRIPTION
**Pull request by duskmoon314**

### CODE REVIEW

**Comment by burrbull** [comment]
Looks like MSRV issues in svd-encoder deps:
https://github.com/bluss/indexmap/issues/234

cc @Emilgardis @therealprof What should we do?

**Comment by Emilgardis** [comment]
Feel free to bump, policy for version bumps is to bump when needed

**Comment by duskmoon314** [comment]
So we need to bump the rust edition to `2021` and the rust version to `1.56.0`?

**Comment by Emilgardis** [comment]
updating edition is not needed, but that sounds good :)

**Comment by duskmoon314** [comment]
A try to bump MSRV to 1.56.0 (2021). Hope not missing something.

File: svd-encoder/Cargo.toml [hunk 1]
```diff
@@ -9,11 +9,12 @@ license = "MIT OR Apache-2.0"
 name = "svd-encoder"
 repository = "https://github.com/rust-embedded/svd"
 edition = "2018"
```
Comment (burrbull) [inline svd-encoder/Cargo.toml:11]: edition

**Review by burrbull — APPROVED** [review]

**Pull request merged**